prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -75,7 +81,7 @@ elements:
     # column C part 2
     # nonstandard
     - name: patient_ssn
-      type: TEXT
+      type: ID_SSN
```

Comment:
Maybe note that we aren't actually collecting this at the moment?
@RickHawesUSDS 's new options stuff is meant to cover this oxymoron situation where a field is required but its also OK to be blank.

Reply:
Please don't put SSN into our data.

Reply:
We won't be. I just wanted to mark the field as what it is, but we don't need to send it through. It can be blank.